epub33/rs/index.html

```diff
@@ -548,6 +548,15 @@ <h3>Spine</h3>
 				<p id="confreq-rs-pkg-spine-unknown" data-tests="#confreq-rs-pkg-spine-unknown">Reading Systems MUST
 					ignore all values expressed in spine <code>itemref</code>
 					<code>properties</code> attributes that they do not recognize.</p>
+
+				<p>
+					When presented with a single <a>Publication Resource</a> that is repeated multiple times in the linear flow 
+					of the spine, Reading Systems SHOULD do their best to display that content in the correct location of that linear flow. 
+					The Reading System SHOULD treat these as distinct pages for UI purposes (for example, each occurrence
+					could be independently bookmarked or annotated), but when following an internal link to that
```

Comment:
"internal link" is a ambiguous when talking about the reading system. Maybe make this clearer by adding the user action: "When a user activates a hyperlink to a resource that is referenced multiple times in the spine, the Reading System SHOULD ..."

Reply:
Agreed "internal link" is not a great choice, but this is less about hyperlink activation and more about resolving the destination of a hyperlink (a RS may not allow following a hyperlink). Maybe:

> When a Reading System follows a hyperlink to a resource referenced multiple times in the spine, the Reading System MUST [...]


That also lets us turn the SHOULD into a MUST.